packages/cli/src/commands/new/test.ts

```diff
@@ -0,0 +1,62 @@
+import BaseCommand from '../../common/base-command'
+import { HasName, ImportDeclaration, joinParsers, parseName } from '../../services/generator/target'
+import { Script } from '../../common/script'
+import Brand from '../../common/brand'
+import { checkCurrentDirIsABoosterProject } from '../../services/project-checker'
+import { generate } from '../../services/generator'
+import { join, resolve } from 'path'
+import { templates } from '../../templates'
+
+const projectName = require(join(resolve(process.cwd()), 'package.json')).name
+
+export default class Test extends BaseCommand {
+  public static description = 'create a new test'
+
+  public static args = [{ name: 'testName' }]
+
+  public async run(): Promise<void> {
+    const { args } = this.parse(Test)
+
+    try {
+      if (!args.testName) throw "You haven't provided a test name, but it is required, run with --help for usage"
+      return run(args.testName)
+    } catch (error) {
+      console.error(error)
+    }
+  }
+}
+
+type TestInfo = HasName
+
+const run = async (name: string): Promise<void> =>
+  Script.init(`boost ${Brand.energize('new:test')} 🚧`, joinParsers(parseName(name)))
+    .step('Verifying project', checkCurrentDirIsABoosterProject)
+    .step('Creating new test', generateTest)
+    .info('Test generated!')
+    .done()
+
+function generateImports(): Array<ImportDeclaration> {
+  return [
+    {
+      packagePath: '@boostercloud/framework-types',
+      commaSeparatedComponents: 'BoosterConfig',
+    },
+    {
+      packagePath: 'chai',
+      commaSeparatedComponents: 'expect',
+    },
+  ]
```

Comment:
Just have a question, does the project's `package.json` includes chai as a dependency? 😮

Reply:
Hey @juanjoman. That is something I mention in PR description "Test files use chai that is not installed by default in new projects." 😄 

I think we should include `chai` as dependency because Booster docs use `chai` but we don't provide it by default? 😅 

+ if `chai` is not in the `project.json` dependencies, what we should use as `expect` assertion instead?

Reply:
@juanjoman I though about importing `expect` from `./expect` if it exists, as we might expect it to be here (because it is [recommended file](http://docs.booster.cloud/chapters/06_testing-booster-applications?id=recommended-files))

If not, check if `chai` exists and import it instead. But if user doesn't have them? 🤔

Reply:
oooh sorry @Fecony, I didn't read it 😓. I think it's ok then to import it from `./expect` and let the user choose their test library 😄